monkey/infection_monkey/system_info/windows_info_collector.py

```diff
@@ -2,14 +2,15 @@
 import logging
 import sys
 
-sys.coinit_flags = 0  # needed for proper destruction of the wmi python module
-
 import infection_monkey.config
 from infection_monkey.system_info.mimikatz_collector import MimikatzCollector
 from infection_monkey.system_info import InfoCollector
 from infection_monkey.system_info.wmi_consts import WMI_CLASSES
 from common.utils.wmi_utils import WMIUtils
 
+
+sys.coinit_flags = 0  # needed for proper destruction of the wmi python module
```

Comment:
I think this needs to happen before the imports, no? Appendix D of `Python Programming on Win32` by Mark Hammond and Andy Robinson for details.

Reply:
As far as I can tell this is used for `pythoncom` module that we don't use, but rolled back just in case.